projects/aca-playwright-shared/src/page-objects/components/dataTable/mat-menu.component.ts

```diff
@@ -49,6 +49,15 @@ export class MatMenuComponent extends BaseComponent {
     await menuElement.waitFor({ state: 'detached' });
   }
 
+  async clickMoreActionsMenuItem(
```

Comment:
we already have method for this  clickMenuItem , we can modify same to add option parameter